sdcm/report_templates/results_longevity.html

```diff
@@ -9,6 +9,44 @@ <h3>
             <li>Instance type: {{ scylla_instance_type }}</li>
             <li>Number of scylladb nodes: {{ number_of_db_nodes }}</li>
 
+            {% if node_benchmarks is not eq("N/A") %}
```

Comment:
Is it a correct comparison? As I can see `node_benchmarks` can be `None` or a dict.

Reply:
I got confused by the `BaseEmailReporter` and it's `build_data_for_report()` method which replaces missing dict values with "N/A". Looks like `{% if node_benchmarks %}` should do it here.